bakery/codec.go

```diff
@@ -85,25 +107,25 @@ func encodeCaveatIdV0(
 		return nil, errgo.Notef(err, "cannot generate random number for nonce")
 	}
 	data := make([]byte, 0, 1+publicKeyPrefixLen+KeyLen+NonceLen+1+binary.MaxVarintLen64+len(rootKey)+len(condition)+box.Overhead)
-	data = append(data, 0) //version
+	data = append(data, byte(Version2)) //version
```

Comment:
It's cleaner and safer doing it this way. You don't need to add all the offsets cumulatively and if you happen to have got the initial size wrong, the only penalty is an allocation, not a panic.